Packs/Campaign/Scripts/GetCampaignIncidentsInfo/GetCampainIncidentsInfo_test.py

```diff
@@ -133,7 +134,7 @@ def test_some_error(mocker):
     # run
     try:
         main()
-        assert False, 'SystemExit should occurred'
+        pytest.fail(msg='SystemExit should occurred')
```

Comment:
If you feel like being a good scout, let's change this to `with pytest.raises(SystemExit): main()`, it's the same but more explicit. Not urgent anyway.